tvnamer/utils.py

```diff
@@ -1045,8 +1053,9 @@ def newPath(self, new_path = None, new_fullpath = None, force = False, always_co
         pointing to the file's new destination if leave_symlink is True.
         """
 
-        if always_copy and always_move:
-            raise ValueError("Both always_copy and always_move cannot be specified")
+        if (always_copy, always_move, always_hardlink).count(True) > 1:
```

Comment:
This would not do the same. I check here if there are more than one value to True. `any` would raise the `ValueError` even with the legit case of one of the always being True.

Reply:
Ahh, quite right! Hopefully this can go away with the enum-based move-mode anyway